code/modules/projectiles/ammo_datums.dm

```diff
@@ -1995,6 +1994,73 @@ datum/ammo/bullet/revolver/tp44
 /datum/ammo/xeno/boiler_gas/corrosive/set_smoke()
 	smoke_system = new /datum/effect_system/smoke_spread/xeno/acid()
 
+/datum/ammo/xeno/redspit
```

Comment:
all of this should be a child of /datum/ammo/xeno/toxin, and just change set_regeants proc

Reply:
done